vhdbuilder/packer/init-variables.sh

```diff
@@ -145,39 +152,27 @@ if [[ "${MODE}" == "linuxVhdMode" ]]; then
 		echo "Using provided SIG_GALLERY_NAME: ${SIG_GALLERY_NAME}"
 	fi
 
-	# Ensure the image-definition name
 	if [[ -z "${SIG_IMAGE_NAME}" ]]; then
-		SIG_IMAGE_NAME=${OS_VERSION//./}
-		if [[ "${OS_SKU}" == "Ubuntu" && "${IMG_SKU}" == "20_04-lts-cvm" ]]; then
-			SIG_IMAGE_NAME=${SIG_IMAGE_NAME}CVM
-		fi
-
-		if [[ "${IMG_SKU}" == *"minimal"* ]]; then
-			SIG_IMAGE_NAME=${SIG_IMAGE_NAME}Minimal
-		fi
-
-		if [[ "${OS_SKU}" == "CBLMariner" ]]; then
-			SIG_IMAGE_NAME=CBLMariner${SIG_IMAGE_NAME}
-		fi
-
-		if [[ "${OS_SKU}" == "AzureLinux" ]]; then
-			SIG_IMAGE_NAME=AzureLinux${SIG_IMAGE_NAME}
-		fi
-
-		if [[ "${ENABLE_TRUSTED_LAUNCH}" == "True" ]]; then
-			SIG_IMAGE_NAME=${SIG_IMAGE_NAME}TL
-		fi
-
-		if [[ "${HYPERV_GENERATION,,}" == "v2" && ("${OS_SKU}" == "CBLMariner" || "${OS_SKU}" == "AzureLinux" || "${OS_SKU}" == "Ubuntu") ]]; then
-			SIG_IMAGE_NAME=${SIG_IMAGE_NAME}Gen2
+		SIG_IMAGE_NAME=$SKU_NAME
```

Comment:
SKU_NAME is set here right? https://github.com/Azure/AgentBaker/blob/master/.pipelines/templates/.builder-release-template.yaml#L54C3-L69C30

is init-variables.sh run after this?

Reply:
yep, init-variables.sh is executed as a part of the `run-packer` make target